clientconn.go

```diff
@@ -1707,15 +1706,15 @@ func (cc *ClientConn) parseTargetAndFindResolver() error {
 		defScheme = resolver.GetDefaultScheme()
 	}
 
-	channelz.Infof(logger, cc.channelz, "fallback to scheme %q", defScheme)
+	logger.Infof("fallback to scheme %q", defScheme)
 	canonicalTarget := defScheme + ":///" + cc.target
 
 	parsedTarget, err = parseTarget(canonicalTarget)
 	if err != nil {
-		channelz.Infof(logger, cc.channelz, "dial target %q parse failed: %v", canonicalTarget, err)
+		logger.Infof("dial target %q parse failed: %v", canonicalTarget, err)
```

Comment:
Seems unnecessary as we will return this error to the user.

Reply:
Good point, deleted.